packages/blocks/src/database-block/common/column-manager.ts

```diff
@@ -72,6 +72,14 @@ class ColumnManager {
     return this.getColumn(type)?.defaultData();
   }
 
+  getHelper(targetType: string) {
+    const column = this.map.get(targetType);
+    if (!column) {
+      throw new Error(`${targetType} is not exist`);
+    }
+    return column;
+  }
+
```

Comment:
Looks the same as this one, maybe you can just refactor this to public
<img width="329" alt="image" src="https://github.com/toeverything/blocksuite/assets/17165520/76580906-e0bb-4d28-abca-253ad4e719e4">